bin/knapsack_pro_queue_rspec_frequently_changing_test_files

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# The purpose of this script is to test that a frequently changing list of test files in Queue Mode
+# can be ordered by ever recorded test files' time execution.
+
+export RAILS_ENV=test
+export KNAPSACK_PRO_ENDPOINT=http://api.knapsackpro.test:3000
+export KNAPSACK_PRO_TEST_SUITE_TOKEN_RSPEC=a28ce51204d7c7dbd25c3352fea222cf
+export KNAPSACK_PRO_CI_NODE_TOTAL=2
+
+# generate an unique commit hash and branch
+export KNAPSACK_PRO_COMMIT_HASH=$(ruby -e "require 'securerandom'; puts SecureRandom.hex")
+export KNAPSACK_PRO_BRANCH=frequently-changing-test-files-$(ruby -e "require 'securerandom'; puts SecureRandom.hex")
+
+echo
+echo
+echo "=== Step 1 ====================================================="
+echo
+echo
+
+# Run all test files in the test suite.
+# Below commands should record time execution for all test files.
```

Comment:
FYI: "Below commands" is incorrect here. It should either be "The below commands" (which might still sound a little unnatural/formal) or "The commands below".

The "below commands" construct is something I've seen you use frequently.